.github/workflows/documentation.yaml

```diff
@@ -37,17 +37,20 @@ jobs:
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           version: 3.x
-      - name: Install Poetry
-        uses: abatilo/actions-poetry@v3
-        with:
-          poetry-version: 1.8.2
-      - name: Configure Poetry
-        run: poetry config virtualenvs.in-project true
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v5
         with:
           python-version: ${{ matrix.python-version }}
-          cache: 'poetry'
+      - name: Setup cache
+        uses: actions/cache@v4
+        with:
+          key: poetry-cache-${{ runner.os }}-${{ matrix.python-version }}-${{ github.job }}
```

Comment:
Do we have to manage the cache manually again?

Is it possible to still use `actions/setup-python` to manage the cache, but add `Install Poetry + Configure` step after?

Reply:
Its best to delete the cache after this is merged, some of the testing packages are using the 3.12 python version

Reply:
I'll delete after this is merged